### PR TITLE
Add `showAllEvents` Calendar Prop

### PR DESCRIPTION
**Pull request by jkhoang313**
This prop will allow the rows in the `month` view to display
all events in a scrollable container, rather than use the
`show more` capability.

### CODE REVIEW

**Comment by jkhoang313** [comment]
@jquense do you mind taking a look at this? The "show + more" messes up the styling for my use of this package so i wanted to provide an alternative.

**Comment by DylanBDev** [comment]
Could really use something like this. I hope this can get reviewed and merged. It looks like exactly what I need. Any tips implementing this before it can be merged?

**Comment by jkhoang313** [comment]
@DylanBDev All you have to do is set `showAllEvents={true}` on the Calendar component and it should be good!

**Review by jquense — COMMENTED** [review]
nice, two small comments thanks! and please run throguh with FF, Chrome, and safari to ensure this is working on all browsers

File: src/Month.js [hunk 1]
```diff
@@ -120,7 +121,7 @@ class MonthView extends React.Component {
         date={date}
         range={week}
         events={events}
-        maxRows={rowLimit}
+        maxRows={showAllEvents ? Infinity : rowLimit}
```
[the remark this answers is not in the document]
Comment (jkhoang313) [reply src/Month.js:124]: Just confirmed that no code assumes this number is bound. It is only used when determining "extra" segments [here](https://github.com/jquense/react-big-calendar/blob/master/src/utils/eventLevels.js#L44)

**Comment by jkhoang313** [comment]
Updated the PR with your suggestions

**Review by jquense — APPROVED** [review]

**Comment by jquense** [comment]
thanks!

**Comment by alxvallejo** [comment]
Most users aren't going to know that they can scroll on a calendar row to reveal more. What i'm trying to get is the Show More button to expand the row to reveal all, but probably just a bit of css needed.

**Pull request merged**